templates/password-reset-requested.twig

```diff
@@ -0,0 +1,14 @@
+{% extends renderHtml ? '_base.html.twig' : '_base.text.twig' %}
+
+{% block content %}
```

Comment:
I drafted this copy, after taking a look at https://postmarkapp.com/guides/password-reset-email-best-practices . Needs someone else to review and/or rewrite.

Reply:
@bdsl This file should be a `.html.twig` file, not a `.twig` file. I made the exact same mistake many months ago and was left scratching my head as to why the email wouldn't fire, until Noel point it out.

Reply:
Good catch, fixed.

Reply:
Nice